runelite-client/src/main/java/net/runelite/client/util/HotkeyListener.java

```diff
@@ -49,6 +49,13 @@ public boolean isEnabledOnLoginScreen()
 		return enabledOnLoginScreen;
 	}
 
+	@Override
+	public void focusLost()
+	{
+		isPressed = false;
+		isConsumingTyped = false;
```

Comment:
this isn't firing the hotkeyReleased() at all, so it wouldn't cause plugins to release their hotkey without them overriding focusLost()?

Reply:
> this isn't firing the hotkeyReleased() at all, so it wouldn't cause plugins to release their hotkey without them overriding focusLost()?

you'd mentioned spurious depress events, maybe I misunderstood but I'd assumed that was related to the hotkeyReleased();